server/models/Notifications.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+
+const ENUM_NOTIFICATIONS = ['Booking', 'Message', 'System', 'Promo'];
+
+const notificationsSchema = new mongoose.Schema({
+    type: {
+        type: String,
+        enum: ENUM_NOTIFICATIONS,
+        required: [true, "Not a valid notification type."],
+    },
+    title: {
+        type: String,
+        required: true,
+    },
+    description: {
+        type: String,
+        required: true,
+    },
+    isRead: {
+        type: Boolean,
+        default: false,
+    },
+    date: {
+        type: String,
+        default: new Date().toDateString(),
+    }
```

Comment:
Wonder if we should add some metadata here? If we wanted to notify a new message or something 🤔 

It would be important to pass in things like urls to view the message or a path to redirect too. Not sure yet though

Reply:
Going through the mockups again it may be useful to include some profile data like a link to the profile picture along with a link to the content source as you mentioned. 

Possibly adding another child schema called `metaData` to store this info could be one way of adding this missing data.

Reply:
Sounds good!